CSI_Greenville_HIST8510_Final_Project-AR.qmd

```diff
@@ -0,0 +1,390 @@
+
+```{r}
+#List all Anderon County deaths in 1870-79 with death type descending
+anderson.seventies <- by.decade.anderson %>%
+  filter(DECADE == "Seventies") %>%
+  select(NAME, DECEASED.DESCRIPTION, DEATH.TYPE) %>%
+  arrange(DEATH.TYPE)
+
+anderson.seventies
+
+```
+
+> There are thirteen accidents tied to a coroner's report in the 1880's in Greenville. There are none listed without surnames, but Julius Brockman is described as a black boy by in the CSI:Dixie data. The thirteen deaths described as natural causes are all recorded as first and last names.
+
+```{r}
+#List all Greenville County deaths in 1880-89 with death type descending
+greenville.eighties <- by.decade %>%
+  filter(DECADE == "Eighties") %>%
+  select(NAME, DECEASED.DESCRIPTION, DEATH.TYPE) %>%
+  arrange(DEATH.TYPE)
+
+greenville.eighties
+```
+
+> Lastly, there are 58 deaths listed as natural causes in Chesterfield in the 1890's. None of the deaths are listed with surnames or described as an African American individual.
+
+```{r}
+#List all Chesterfield County deaths in 1890-99 with death type descending
+chesterfield.nineties <- by.decade.chesterfield %>%
+  filter(DECADE == "Nineties") %>%
+  select(NAME, DECEASED.DESCRIPTION, DEATH.TYPE) %>%
+  arrange(DEATH.TYPE)
+
+chesterfield.nineties
+```
+
+**Conclusion**
+
+Although there is a great deal to be learned from assembling the coroner inquests that remain from the 19th century in South Carolina, in limiting my focus to a few counties that represent the different regions of that state it seems clear to me that the deaths directly coincide with the economies of these regions. Accidents and natural causes are the leading causes of death in Greenville, Anderson and Chesterfield during their peak decades of recorded coroner inquests. Accidents, especially industrial and vehicle accidents, are the top listed causes of death in the Piedmont region in the 1870-80s. This directly corresponds to the years after the Civil War and the rebuilding of a more industrialized economy. The natural deaths in Chesterfield in the 1890s speaks to the lack of industry and rural nature of that region, especially after the devastation of the Civil War. Horry County is a bit of an outlier, but again, examining its early economic development the rate of homicides during the 1850s and the relatively high percentage of black deaths, this can be seen through the lens of plantation economics as well as the slave uprisings in the days leading up to Civil War.
+
+**Bibliography**
+
+Berry, Stephen. *Count the Dead: Coroners, Quants, and the Birth of Death As We Know It.* Chapel Hill, North Carolina: University of North Carolina Press, 2022.
```

Comment:
See this website for an overview of how to do footnotes in R: https://www.markdownguide.org/